app/views/refinery/_footer.html.erb

```diff
@@ -1,39 +1,57 @@
-<div class="container">
-  <div class="footer-top">
-    <div class="link-section footer-about-links">
-      <span class="link-category-heading"><%= t('footer.about_mapc') %></span>
-      <ul>
-        <li><a href="https://www.mapc.org/about-mapc/staff/"><%= t('footer.staff') %></a></li>
-        <li><a href="https://www.mapc.org/get-involved/metrofuture-our-regional-plan/">MetroFuture</a></li>
-      </ul>
-    </div>
-    <div class="link-section other-links">
-      <div class="top-links">
-        <span class="link-category-heading"><%= t('footer.receive_updates') %></span>
-        <div>
-          <a class="button" href="<%= CONSTANT_CONTACT_LANDING_PAGE %>" target="_blank"><%= t('footer.join_our_email_list') %></a>
-        </div>
+<footer class="footer">
+  <div class="container">
+    <div class="footer__row">
+      <div class="footer__column">
+        <h2><%= t('footer.about_mapc') %></h2>
+        <p>The Metropolitan Area Planning Council (MAPC) is the regional planning agency serving the people who live and work in the 101 cities and towns of Metropolitan Boston.</p>
       </div>
-      <div class="bottom-links">
-        <a href="<%= (Mobility.locale != Refinery::I18n.config.default_locale ? "/#{Mobility.locale.to_s}" : "") %>/privacy-policy"><%= t('footer.privacy_policy') %></a>
+      <div class="footer__column">
+        <h2>Quick Links</h2>
+        <ul>
+          <li><a href="/">Home</a></li>
+          <li><a href="/process-details">Process Details</a></li>
+          <li><a href="/events">Events</a></li>
+          <li><a href="<%= (Mobility.locale != Refinery::I18n.config.default_locale ? "/#{Mobility.locale.to_s}" : "") %>/privacy-policy"><%= t('footer.privacy_policy') %></a></li>
+        </ul>
+      </div>
+      <div class="footer__column">
+        <h2>Explore</h2>
+        <ul>
+          <li><a href="https://www.mapc.org/about-mapc/staff/">Staff</a></li>
+          <li><a href="https://www.mapc.org/get-involved/metrofuture-our-regional-plan/">MetroFuture</a></li>
+          <li><a href="https://datacommon.mapc.org/">DataCommon</a></li>
+          <li><a href="https://www.massbuilds.com/">MassBuilds</a></li>
+          <li><a href="https://trailmap.mapc.org/">Trailmap</a></li>
+        </ul>
```

Comment:
Should we spawn a new tab for these external sites? 

`<a ... rel="noopener noreferrer" target="_blank">..</a>`

Reply:
Sure we can do that.